readme.md

```diff
@@ -181,6 +181,27 @@ queryString.parse('foo=true', {parseBooleans: true});
 
 Parse the value as a boolean type instead of string type if it's a boolean.
 
+##### types
+
+Type: `object`\
+Default: `{}`
+
+```js
+const queryString = require('query-string');
+
+queryString.parse('foo=1&bar=1&baz=1.5', {
+    types: {
+        foo: 'string',
+        bar: 'number',
```

Comment:
What about an array of string or array of numbers? We should probably use the notation `string[]` and `number[]` for those.

Reply:
Agree